### PR TITLE
Updated Mixed content test to reflect new behavior

### PR DESCRIPTION
**Pull request by pgagne**
Having mixed content types no longer is an issue, the non-yum content types will not be exported.

Changed test_negative_export_cv_with_mixed_content_repos to test_postive_export_cv_with_mixed_content_repos

See https://bugzilla.redhat.com/show_bug.cgi?id=1726457

### CODE REVIEW

**Comment by pgagne** [comment]
Just wanted to mention there are other testcases that were affected by changed behavior with respect to CVV with yum repos that aren't set to "immediate", I plane to update those seprately as part of the closed loop tests for https://bugzilla.redhat.com/show_bug.cgi?id=1671319

File: tests/foreman/cli/test_satellitesync.py [hunk 1]
```diff
@@ -408,6 +408,23 @@ def tearDown(self):
         super(ContentViewSync, self).tearDown()
         ssh.command('rm -rf {}'.format(self.export_dir))
 
+    def assert_exported_cvv_exists(self, content_view_name, content_view_version):
+        """Verfiy an exported tar exists
```
Comment (omkarkhatavkar) [inline tests/foreman/cli/test_satellitesync.py]: `Verfiy` typo here?

File: tests/foreman/cli/test_satellitesync.py [hunk 2]
```diff
@@ -1160,7 +1177,7 @@ def test_negative_export_cv_with_puppet_repo(self):
         )
 
     @tier3
-    def test_negative_export_cv_with_mixed_content_repos(self):
+    def test_postive_export_cv_with_mixed_content_repos(self):
         """Exporting CV version having yum and non-yum(puppet) repos throws error
```
Comment (omkarkhatavkar) [inline tests/foreman/cli/test_satellitesync.py]: If the intention of the test has changed from negative to positive, I'm assuming you will not be required say 'throws error' ? Please update the docstring with valid intent.

File: tests/foreman/cli/test_satellitesync.py [hunk 3]
```diff
@@ -408,6 +408,23 @@ def tearDown(self):
...
+
+        :return: The path to the tar (if it exists).
+        """
+        exported_tar = '{0}/export-{1}-{2}.tar'.format(
+            self.export_dir, content_view_name, content_view_version)
+        result = ssh.command("[ -f {0} ]".format(exported_tar))
+        self.assertEqual(result.return_code, 0)
+        return exported_tar
+
+    def assert_import_export_packages_match(self, importing_cvv_id, exporting_cvv_id):
```
Comment (omkarkhatavkar) [inline tests/foreman/cli/test_satellitesync.py]: I don't know where are you consuming this function?

Comment (pgagne) [reply tests/foreman/cli/test_satellitesync.py]: Hmm, I think I added this with the intent of using it in future tests, I didn't end up using it in this test, but probably will in the future. Can we keep it?

**Comment by pgagne** [comment]
test results 

```
+ /home/[*******]/shiningpanda/jobs/ad0ef6b5/virtualenvs/d41d8cd9/bin/py.test -v --junit-xml=foreman-results.xml -o junit_suite_name=standalone-automation -m 'not stubbed' tests/foreman/cli/test_satellitesync.py -k test_postive_export_cv_with_mixed_content_repos
============================= test session starts ==============================
platform linux -- Python 3.6.6, pytest-4.6.3, py-1.8.0, pluggy-0.13.1 -- /home/[*******]/shiningpanda/jobs/ad0ef6b5/virtualenvs/d41d8cd9/bin/python3.6
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/[*******]/workspace/satellite6-standalone-automation
plugins: xdist-1.25.0, services-1.3.1, mock-1.10.4, forked-1.1.3, env-0.6.2
collecting ... 2019-12-03 15:53:21 - conftest - DEBUG - Collected 61 test cases

2019-12-03 10:53:22 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f59f1d67438
2019-12-03 10:53:22 - robottelo.ssh - INFO - Connected to [ibm-x3250m4-06.lab.eng.rdu2.redhat.com]
2019-12-03 10:53:22 - robottelo.ssh - INFO - >>> rpm -q satellite
2019-12-03 10:53:23 - robottelo.ssh - INFO - <<< stdout
satellite-6.7.0-4.beta.el7sat.noarch

2019-12-03 10:53:23 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f59f1d67438
2019-12-03 10:53:23 - robottelo.host_info - DEBUG - Host Satellite version: 6.7
2019-12-03 10:53:23 - robottelo.helpers - DEBUG - Generated file bz_cache.json with BZ collect data
collected 61 items / 60 deselected / 1 selected

tests/foreman/cli/test_satellitesync.py::ContentViewSync::test_postive_export_cv_with_mixed_content_repos PASSED [100%]
```

**Pull request merged**